test-basic/service-caller.js

```diff
@@ -70,7 +70,8 @@ describe('Service caller', function() {
             });
     });
 
-    it('postOfUrlencodedForDocumentArray1 endpoint', function(done) {
+    // errors all the time now, should fix.
```

Comment:
This is passing consistently on Jenkins - https://ml-clt-jenkins.progress.com/job/devexp/job/Node-Client/job/Node-client-api/job/develop/160/ - so maybe just a local issue? Does it fail when you run all of "test-basic"?